license.txt

```diff
@@ -1,6 +1,7 @@
-Copyright 2022 diwako
+Original Copyright 2020-2022 diwako
+Adaptions 2022 Snick0rz/Celene 
```

Comment:
You can add a new section in the read me for contirbutors. I 100% want people that contributed to be credited. license is just the default legal blabla usually.